### PR TITLE
update alumni

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

discussed @yukw777 moving as Alumni but happy to welcome him back as core when he feels so :]

## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [ ] Did you write any new necessary tests? (not for typos and docs)
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: docs/source/governance.rst [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-PyTorch Lightning Governance | Persons of interest
-==================================================
+Lightning Governance | Persons of interest
+==========================================
 
 Leads
 -----
 - William Falcon (`williamFalcon <https://github.com/williamFalcon>`_) (Lightning founder)
-- Jirka Borovec (`Borda <https://github.com/Borda>`_)
```
Comment (awaelchli) [inline docs/source/governance.rst:9]: it's a good thing, right? 😄

File: docs/source/governance.rst [hunk 2]
```diff
@@ -1,12 +1,12 @@
 .. _governance:
 
-PyTorch Lightning Governance | Persons of interest
-==================================================
+Lightning Governance | Persons of interest
+==========================================
```
Comment (awaelchli) [inline docs/source/governance.rst:4]: Why delete? The name was accurate or not?

**Review by awaelchli — APPROVED** [review]

File: docs/source/governance.rst (hunk 2 above)
Comment (Borda) [reply docs/source/governance.rst:4]: it was too long in the left menu

File: docs/source/governance.rst (hunk 1 above)
Comment (Borda) [reply docs/source/governance.rst:9]: you never know :D

File: docs/source/governance.rst (hunk 2 above)
Comment (kaushikb11) [reply docs/source/governance.rst:4]: But we have multiple Lightning projects now 😄

File: docs/source/governance.rst [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 
 Leads
 -----
 - William Falcon (`williamFalcon <https://github.com/williamFalcon>`_) (Lightning founder)
-- Jirka Borovec (`Borda <https://github.com/Borda>`_)
+- Jirka Borovec (`Borda <https://github.com/Borda>`_) (longest standing)
```
Comment (Borda) [inline docs/source/governance.rst:9]:
```suggestion
- Jirka Borovec (`Borda <https://github.com/Borda>`_)
```

**Review by s-rog — APPROVED** [review]

**Review by ananthsub — APPROVED** [review]

**Review by SkafteNicki — APPROVED** [review]

**Pull request merged**